### ISSUE TITLE
Reset progress

### ISSUE THREAD

**Issue by askl56** [Codewars/codewars.com#199]
There should be a button to remove account progress or remove account

**Comment by bkaestner** [comment]
Possible duplicate of #140.

**Comment by sotojuan** [comment]
@bkaestner This is not a duplicate at the moment. I deleted my account months ago. I signed in through GitHub again (it let me as if I was a real, regular user) and all my points and stats were still there. I'd like a way to actually delete my account for real along with the data.

**Comment by egomadking** [comment]
Bump. I just changed from focusing on JS to Ruby. I would prefer it if my level could be reset to reflect my change in focus.

**Comment by expertchi** [comment]
+ For this , user should have option to start their progress over without deleting their account

**Comment by bountonw** [comment]
I agree. The inability to reset progress has led me to look for alternatives. I will try back later.

**Comment by jonnydh** [comment]
Agreed, would be a big help.

**Comment by royemosby** [comment]
I found that deleting my account and starting with a new one worked out just fine. I had both my ex-account and my new account associated with GitHub and there were no problems getting set back, in minutes, up with 0 progress.
